### PR TITLE
Added integration tests, readme details andconfigured eureka properly

### PR DESCRIPTION
**Pull request by R4il**

### CODE REVIEW

File: src/test/java/org/aist/aide/mappingservice/integration/MappingServiceIntegrationTests.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+@SpringBootTest
+@ActiveProfiles(profiles = "test")
+public class MappingServiceIntegrationTests {
+
+    @Autowired
+    private MappingController mappingController;
+
+    @After
+    public void tearDown() {
+        mappingController.deleteMapping(1);
```
Comment (Castyr) [inline src/test/java/org/aist/aide/mappingservice/integration/MappingServiceIntegrationTests.java]: Won't these keys be different after each test? Are they not auto-incrementing?

**Review by Castyr — APPROVED** [review]
Just check my one comment.

File: src/test/java/org/aist/aide/mappingservice/integration/MappingServiceIntegrationTests.java (hunk 1 above)
Comment (R4il) [reply src/test/java/org/aist/aide/mappingservice/integration/MappingServiceIntegrationTests.java]: Oh, yeah I shouldnt have added that. I did that at one point checking. something. In the end I have no teardown.

**Comment by R4il** [comment]
[Finish up tests and eureka for mapping services.](https://trello.com/c/wQSBso8G/82-finish-up-tests-and-eureka-for-mapping-services)

**Pull request merged**